java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -994,6 +1021,23 @@ public final ColumnBuilder append(boolean value) {
       return this;
     }
 
```

Comment:
From a performance standpoint when going from a row to a column we want to do the minimal processing in between. Spark supports an API that would let us read out an unscaled long value from the `Decimal` if it is small enough to not need a full BigDecimal, which are the only types that we would be able to support.  I really would like to see an API that would let us do this more efficiently, and all it would do is check that you got the appropriate decimal type.  It would be much much faster.

```
public final ColumnBuilder appendUnscaledDecimal(long value)
public final ColumnBuilder appendUnscaledDecimal(int value)
```

Reply:
Good idea! I've added primitive-oriented append methods, as below:  
```java
public final ColumnBuilder appendUnscaledDecimal(long value)
public final ColumnBuilder appendUnscaledDecimal(int value)
public Builder appendUnscaledDecimalArray(int... values)
public Builder appendUnscaledDecimalArray(long... values)
```